src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -48,6 +48,8 @@ def load_command_table(self, _):
         g.custom_command('show-deploy-log', 'app_get_build_log')
         g.custom_command('set-deployment', 'app_set_deployment',
                          supports_no_wait=True)
+        g.custom_command('unset-deployment', 'app_unset_deployment',
```

Comment:
the commands quite confusing:
```
az spring-cloud app deployment create
az spring-cloud app set-deployment
az spring-cloud app unset-deployment
```

To consistent, why not
```
az spring-cloud app deployment set --type <production/non-production>?
```

Reply:
Since they have different parameters, and "set-deployment" has been used by customers, hope not to introduce breaking change.